### DISCUSSION TITLE
Deploy Wazuh agent on Window endpoint

### DISCUSSION THREAD

**Discussion by bachlx19992** [wazuh/wazuh#19511] [Q&A]
Hi everyone,
I am deploying wazuh agent to the window server according to the instructions https://documentation.wazuh.com/4.4/installation-guide/wazuh-agent/wazuh-agent-package-windows.html
But Wazuh server reported an error Could not fetch data for this agent 
Has anyone ever had it? and how to solve the problem?
Please help me, many thanks.

**Comment by sellarsj** [comment]
windows firewall. allow the required ports.
